src/main/java/io/bf2/kafka/authorizer/CustomAclAuthorizer.java

```diff
@@ -422,22 +423,17 @@ public List<CompletionStage<AclCreateResult>> createAcls(AuthorizableRequestCont
                 final CompletionStage<AclCreateResult> result;
 
                 if (!binding.entry().principal().startsWith(CustomAclBinding.USER_TYPE_PREFIX)) {
```

Comment:
Can we pull `binding.entry().principal()` out as a variable to reduce the noise in the code.

Reply:
done